sn_transfers/src/transfers/unsigned_transaction.rs

```diff
@@ -93,11 +111,21 @@ impl UnsignedTransaction {
             })
             .collect();
 
+        // order inputs by value, re const after sorting
+        let mut cashnotes_big_to_small = available_cash_notes;
```

Comment:
we shall try to use the small ones first ?

Reply:
Ideally we'd like to minimize the amount of Spends created, using big ones first has that effect for now, but it's not optimal over the long run. 

In the future I imagine we should create a better algorithm that would optimize that and choose the best options with the closest values as to minimize the amount of spends and avoid using huge Cashnotes for small Txs. 

Also, since CashNotes don't have a determined value anymore, we could return the change value to an existing unspent CashNote that would accumulate wealth until it is spent. 

All these nice optimizations will not change the API in this PR so that's also nice!